test/services/auth.test.ts

```diff
@@ -2,59 +2,129 @@
  * @jest-environment node
  */
 
-import { authenticateLogin } from '../../src/services/auth';
+import { User } from '@prisma/client';
+
+import { authenticateLogin, authenticateGoogleLogin } from '../../src/services/auth/auth';
 import { user } from '../factories/user';
-import { createUser } from '../../src/app/actions/user';
+import { createUser } from '@/app/actions/user';
+import * as userRepository from '@/repositories/user';
+import * as identityRepository from '@/repositories/identity';
+import { AuthProviders } from '@/constants/enum';
 
 describe('Auth:', () => {
-  describe('given correct credentials', () => {
-    it('returns user', async () => {
-      const userPayload = { name: user.name, email: user.email, password: user.password };
-
-      await createUser(userPayload);
-      const data = await authenticateLogin({
-        email: user.email,
-        password: user.password,
+  describe('Authenticate credential login', () => {
+    describe('given correct credentials', () => {
+      it('returns user', async () => {
+        const userPayload = { name: user.name, email: user.email, password: user.password };
+
+        await createUser(userPayload);
+        const data = await authenticateLogin({
+          email: user.email,
+          password: user.password,
+        });
+
+        expect(data).not.toBeNull();
+        expect(data.email).toEqual(user.email);
+        expect(data.name).toEqual(user.name);
       });
+    });
 
-      expect(data).not.toBeNull();
-      expect(data.email).toEqual(user.email);
-      expect(data.name).toEqual(user.name);
+    describe('when there is no user as credential', () => {
```

Comment:
```suggestion
    describe('given there is no user as credential', () => {
```

As you are using `given` many places